back/lib/participation_method/voting.rb

```diff
@@ -4,6 +4,20 @@ module ParticipationMethod
   class Voting < Ideation
     def assign_defaults_for_participation_context
       participation_context.ideas_order ||= 'random'
+      participation_context.voting_max_votes_per_idea = 1 if participation_context.voting_method == 'single_voting'
```

Comment:
Since we're checking for equality of the voting_method with 'single_voting' and 'multiple_voting', it could make sense to delegate this logic to the voting method. But this is fine for me as well.